include/cuda/std/detail/libcxx/include/span

```diff
@@ -32,42 +39,39 @@ template <class ElementType, size_t Extent>
         (sizeof(ElementType) * Extent))> as_writable_bytes(span<ElementType, Extent> s) noexcept;
 
 
-namespace std {
 template <class ElementType, size_t Extent = dynamic_extent>
 class span {
 public:
     // constants and types
     using element_type = ElementType;
     using value_type = remove_cv_t<ElementType>;
-    using index_type = size_t;
+    using size_type = size_t;
```

Comment:
I'm surprised by the changes to the synopsis docs. Was `span` implementation in libc++ out of date or something?

Reply:
Yes, there were most notably

- P1872R0 span Should Have size_type, Not index_type
- P2116R0 Removing tuple-Like Protocol Support From Fixed-Extent span

I dont remember which one removed the const_iterators